src/components/List.js

```diff
@@ -127,6 +144,7 @@ const List = ({ token }) => {
                     disabled={checkPurchasedDate(doc.lastPurchased)}
                   />
                   {doc.itemName}
+                  <button onClick={() => confirmDelete(doc)}>Delete</button>
```

Comment:
While the project wireframe shows `delete` as a link, the resources we found advised the use of a button. 
[When to Use Buttons and Links](https://balsamiq.com/learn/articles/buttons-links/)
```
Links should never be used to change the state of an application. This means that clicking on a link should not add, change, or delete any data on the screen, So, a delete action should never be a link, nor should a save action.
```

Reply:
agreed

Reply:
Great choice!